api/views.py

```diff
@@ -60,6 +68,80 @@ def delete(self, request):
         return response.Response(status=status.HTTP_200_OK)
 
 
+# send email..
+class UserForgotPassword(views.APIView):
+    permission_classes = (permissions.AllowAny,)
+
+    def post(self, request):
+        username = request.data["username"]
+        # check if email exists
+        if User.objects.filter(username=username).exists():
+            # encode userId
+            currentUser = User.objects.get(username__exact=username)
+            uidb64 = urlsafe_base64_encode(
+                force_bytes(currentUser.pk)
+            )  # encoded userId
+            # get domain we are on
+            domain = get_current_site(request).domain
+            # relative url for verification
+            link = reverse(
+                "activate",
+                kwargs={
+                    "uidb64": uidb64,
+                    "token": account_activation_token.make_token(currentUser),
+                },
+            )
+            reset_url = "http://" + domain + link
+            try:
+                send_mail(
+                    "Password reset request",
+                    "Hi "
+                    + currentUser.username
+                    + " Please use this link to reset your account\n"
+                    + reset_url,
+                    "olawinridwan18@gmail.com",  # Need to change to a conteo email
```

Comment:
Is this parameter the email it is sent from?

Reply:
Yes.

Reply:
Might as well use our 7th member